recipe/meta.yaml

```diff
@@ -176,7 +176,6 @@ about:
   dev_url: https://git.ligo.org/lscsoft/ligo.skymap
   license: GPL-3.0-or-later
   license_family: GPL
-  license_file: LICENSE.rst
```

Comment:
I think we want to keep this line, so that conda build knows what to look for inside the distribution. Deleting the duplicate is of course correct.